From 1acb290624dc6a5b266e58999f09275b400166f8 Mon Sep 17 00:00:00 2001
From: devoto <vastavale@inwind.it>
Date: Mon, 1 Nov 2010 09:07:06 +0100
Subject: [PATCH] Ciao provo

---
 README |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/README b/README
index 5a93db1..8ccd901 100644
--- a/README
+++ b/README
@@ -1 +1,3 @@
-ghedi resto
\ No newline at end of file
+Domani vado al lavoro
+
+ghedi resto
-- 
1.7.3.1

